container-images/env-inject/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
This is cool. I am going to see if there might be a way to do this directly from within `webpack` so we wouldn't need this at all.